bin/ghe-backup

```diff
@@ -197,8 +197,18 @@ ghe-backup-audit-log || failures="$failures audit-log"
 echo "Backing up hookshot logs ..."
 ghe-backup-es-hookshot || failures="$failures hookshot"
 
+# if we should use gitbackups to backup repositories
+should_use_gitbackups_for_repositories(){
```

Comment:
we need to think through this a bit more, what if the script isn't on the machine for instance (or is this protected against in some other way?)